src/components/Button.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Button(props) {
+  const { name, color, wide } = props;
+
+  return (
+    <button
+      type="button"
+      className={`${color} ${wide ? 'twice' : ''}`}
+    >
+      {name}
+    </button>
+  );
+}
+
+Button.propTypes = {
+  name: PropTypes.string,
+  color: PropTypes.string,
+  wide: PropTypes.bool,
+};
+
+Button.defaultProps = {
+  name: 'Button',
+  color: '',
```

Comment:
**REQUIRED**
* The button default color should be orange unless changed by a color prop([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/colorProps.png) from